test/fixtures/preact.js

```diff
@@ -346,7 +346,7 @@
         return dom;
     }
     function unmountComponent(component) {
-        if (options.beforeUnmount) options.beforeUnmount(component);
+        if (options.unmount) options.unmount(component);
```

Comment:
Looks like this is a compiled version of `Preact 8.x`.

Reply:
Removed that change 🎉

Reply:
ack!